embedded-maven/README.txt

```diff
@@ -0,0 +1,3 @@
+aar is a sample maven project to deal with aar
```

Comment:
Better to use .md format for the README file, which is expected to briefly describe what the sample wants to show and how end user to use it.